server/startup/migrations/v212.js

```diff
@@ -0,0 +1,21 @@
+import { MongoInternals } from 'meteor/mongo';
+
+import { Migrations } from '../../../app/migrations';
+
+Migrations.add({
+	version: 212,
+	async up() {
+		const { mongo } = MongoInternals.defaultRemoteCollectionDriver();
+		const apps = mongo.db.collection('rocketchat_apps');
+
+		await apps.update({
```

Comment:
removing `async` from function, you can use `Promise.await` here, so it will await using Fibers :wink:

Reply:
done. thanks for the tip